eip-template.md

```diff
@@ -1,120 +1,126 @@
 ---
```

Comment:
```suggestion
---
eip: 7563
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.